src/pages/RepositoryPage.tsx

```
@@ -10,26 +12,44 @@ interface RepositoryPageProps {
 }
 
-export function RepositoryPage({ onLogout, onLogin, isAuthenticated }: RepositoryPageProps) {
+function RepositoryPageContent({ onLogout, onLogin, isAuthenticated }: RepositoryPageProps) {
   const params = useParams<{ owner?: string; repo?: string }>();
 
-  const searchQuery = buildSearchQuery(params);
+  // Load organizations only if authenticated and on /orgs route
+  let organizations: Array<{ login: string }> = [];
+  if (isAuthenticated && !params.owner && !params.repo) {
+    try {
+      const { organizations: orgs } = useViewer();
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)